### DISCUSSION TITLE
Drop tables completely

### DISCUSSION THREAD

**Discussion by ultimoamore** [pgpartman/pg_partman#431] [Q&A]
Hi,

I would need to completely drop a table managed by pg_partman (native partitioning).
I don't want to keep any data.
Is there a fast way of doing it?
Using undo_partition data I need to create a "target table" even if I'm not interested in data...

**Comment by keithf4** [comment]
If you just want to remove the table in its entirety and not keep any data, you can just drop the parent table (which should drop the children) and then delete the entry from the `part_config` table via a normal `DELETE ... WHERE parent_table = ...` statement

**Reply by ultimoamore** [reply]
And I suppose also a drop table for the template table?
It seems like a workaround... maybe a flag in undo_partition?

**Reply by keithf4** [reply]
Yes, sorry the template table as well. 

I may see about an option for undo_partitioning. However I considered that more that you're returning a partitioned table to a non-partitioned table, not so much getting rid of it. So maybe a completely different function all together.
